src/index.html

```diff
@@ -4,11 +4,195 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Title</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/meyer-reset/2.0/reset.min.css">
     <link rel="stylesheet" href="./styles/main.css">
   </head>
   <body>
-    <img src="./images/logo.svg" alt="Logo">
-    <h1>Hello Mate Academy</h1>
-    <script type="text/javascript" src="scripts/main.js"></script>
+    <header class="header">
+      <div class="header__container">
+        <img src="images/Rectangle 2.png" alt="airbnb-logo" class="img--heigth">
+        <nav class="nav">
+          <ul class="nav__items">
+            <li class="nav__item--indent text-size16">
+              <a href="#" class="link">Become a host</a>
+            </li>
+            <li class="nav__item--indent text-size16">
+              <a href="#" class="link">Trips</a>
+            </li>
+            <li class="text-size16">
+              <a href="#" class="link">Messages</a>
+            </li>
+          </ul>
+          <a href="#" class="img_heigth">
+            <img src="images/Ellipse.png" alt="profile-link" class="img__profile img_heigth">
+          </a>
+        </nav>
+
+        <nav class="menu__hamburger">
+          <input id="menu__toggle" type="checkbox" />
+          <label class="menu__btn" for="menu__toggle">
+            <span></span>
+          </label>
+          <div class="menu__box">
+                  <img src="images/Ellipse.png" alt="profile-link" class="menu__item_img img__profile img_heigth">
+              <ul>
+                  <li class="menu__item text-size16">
+                    <a href="#" class="link">Become a host</a>
+                  </li>
+                  <li class="menu__item text-size16">
+                    <a href="#" class="link">Trips</a>
+                  </li>
+                  <li class="menu__item text-size16">
+                    <a href="#" class="link">Messages</a>
+                  </li>
+                </ul>
+
+          </div>
+
+        </nav>
+      </div>
+    </header>
+
+    <section class="images__main-container">
+      <img src="images/Image.png" alt="apartment-images" class="images__main">
+    </section>
+
+    <div class="container">
+      <main class="main">
+
+          <section class="settlement_media">
+              <div class="settlement-form">
+                <p class="settlement__prise-night text-size18">25$ per night</p>
```

Comment:
wrong using of BEM `settlement__prise-night`
where is the class `settlement`?

Reply:
btw, what is `prise`?

Reply:
it's demonstrated my excellent english